nss-day-cohort-47/holiday-road-tundra-travelers
# cc-styles

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
Just prettier, I cleaned up the page and several unneeded lines.
## Type of change
Cosmetic and clerical

Please delete options that are not relevant.


# Testing Instructions

Use your beautiful eyes to witness the difference.
Also open the nav bar to see the copyright is in there now.


# Checklist:
